terrain/voxel_lod_terrain.cpp

```diff
@@ -83,6 +83,13 @@ VoxelLodTerrain::~VoxelLodTerrain() {
 	}
 }
 
+String VoxelLodTerrain::get_configuration_warning() const {
```

Comment:
Warning when a part of the data will not get represented is interesting, but the design of how this information gets detected would probably need more thought

Reply:
You can see the only current use case that generates a warning is VLT + Type which results in a blank screen for the user. That's likely the cause behind #110 and has certainly caused me and others a lot of headaches.

Reply:
I had that too when doing blocky stuff at some point so it's a good idea, just needs different source for the info

Reply:
Now it only queries when they are generators.

Reply:
I think it's better if `VoxelStream` has a function to get a mask of used channels, rather than limiting it to a single channel on generators only.

Reply:
`VoxelStream` now has nothing to do with channels. That was moved to `VoxelGenerator`. Do you want to keep it like this?

Instead of get_channel returning an enum, you want it to return an 8-bit mask?

Then I could have the enum values be 0x1, 0x2, 0x4, 0x8 instead of 1, 2, 3, 4. Then the bits and enum will still work together.  Then the VLT check still remains essentially the same as it warns on all channels except for SDF at the moment. I'll start working on this.

Reply:
Channel values can remain the same, they only turn as bitmask here. Conversion is straightforward, `1 << channel` to obtain mask from single channel.